site/app/controllers/admin/UsersController.php

```diff
@@ -608,6 +608,29 @@ public function updateRotatingSections() {
             $this->core->getQueries()->updateUsersRotatingSection($i + 1, $update_users);
         }
 
+        //update graders' access for gradeables with all access grading for limited
+        //access graders now that rotating sections are set up
+
+        //$update_graders_gradeables is all gradeables in this semester/course where
+        //there's limited access graders + all access grading
+        $update_graders_gradeables = [];
+        $update_graders_gradeables_ids = $this->core->getQueries()->getLimitedGradersAllAccessGradeablesIds();
+        foreach ($update_graders_gradeables_ids as $row) {
+            $g_id = $row['g_id'];
+            $tmp_gradeable = $this->tryGetGradeable($g_id, false);
+            if ($tmp_gradeable === false) {
+                continue;
+            }
+            array_push($update_graders_gradeables, $tmp_gradeable);
```

Comment:
Use the `[]` syntax for pushing to an array with a single element as it's a bit quicker (as there's no overhead of a function call). So for here, it would be:
```
$tmp_gradeable[] = $this->tryGetGradeable($g_id, false);
```